### PR TITLE
BitGo support

### PR DESCRIPTION
**Pull request by yivo**

### CODE REVIEW

**Comment by yivo** [comment]
@mkuvshynnikov I have good progress on it. I think tomorrow I will finish this PR.

File: lib/coin_api/bit_go.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+module CoinAPI
+  class BitGo < BaseAPI
+    def initialize(*)
+      super
+      @endpoint     = 'https://test.bitgo.com/api/v2/t' + currency.code.downcase
```
Comment (mod) [inline lib/coin_api/bit_go.rb]: We are going to need a VAR for this parameter

File: app/models/worker/deposit_coin.rb [hunk 2]
```diff
@@ -4,32 +4,26 @@ class DepositCoin
...
 
         tx = PaymentTransaction::Normal.create! \
-          txid: txid,
-          txout: txout,
-          address: detail[:address],
-          amount: detail[:amount].to_s.to_d,
-          confirmations: raw[:confirmations],
-          receive_at: Time.at(raw[:timereceived]).to_datetime,
+          txid: tx[:id],
+          txout: 1,
```
Comment (mod) [inline app/models/worker/deposit_coin.rb]: what was the value of txout ?

**Review by mod — CHANGES_REQUESTED** [review]

File: lib/coin_api/bit_go.rb (hunk 1 above)
Comment (yivo) [reply lib/coin_api/bit_go.rb]: @mod Sure. I put it for the WIP period of patch.

File: app/models/worker/deposit_coin.rb (hunk 2 above)
Comment (yivo) [reply app/models/worker/deposit_coin.rb]: @mod It was used as index in list of `transaction[:details]`. This list always contains one entry of `payment` category.

**Pull request merged**